### ISSUE TITLE
Read a TV m3u8 with videoJS

### ISSUE THREAD

**Issue by Impre-visible** [videojs/video.js#7985]
Hi guys, i'm french, and in France, if our tv provider is Freebox, we have access to our own m3u8 playlist with all link to the tv. I want to have an ui to select the tv channel, I see videojs-playlist, a plugin for videojs, but I don't see if I can use that with a m3u8 playlist of multiples m3u8, do you have an idea how can I do that ? Thanks

**Comment by gkatsev** [comment]
videojs-playlist is pretty simple, it's an array of sources. These sources can be m3u8s, or anything that videojs can play.

**Comment by Impre-visible** [comment]
> videojs-playlist is pretty simple, it's an array of sources. These sources can be m3u8s, or anything that videojs can play.

Yes, but my list of m3u8s is a m3u8, so i have to parse with js the m3u8 ? I can't directly import a m3u8 as a list of video stream ?

**Comment by gkatsev** [comment]
Oh, I see, I didn't understand what you meant. No, that plugin doesn't support that. It needs an array of playlist item objects, see https://github.com/videojs/videojs-playlist/blob/main/docs/api.md#playlist-item-object

You should be able to `fetch()` your m3u8 and pretty easily convert it into that object that videojs-playlist expects.
